### ISSUE TITLE
Email prompt hangs on `balena login` credentials menu

### ISSUE THREAD

**Issue by pdcastro** [balena-io/balena-cli#1322]
- **balena CLI version:** e.g. 11.3.3 and other 11.x.x versions too
- **Operating system version:** All (Windows, Linux, macOS)
- **32/64 bit OS and processor:** 64-bit Intel
- **Install method:** standalone zip

```
$ balena login
 _            _
| |__   __ _ | |  ____  _ __    __ _
| '_ \ / _` || | / __ \| '_ \  / _` |
| |_) | (_) || ||  ___/| | | || (_) |
|_.__/ \__,_||_| \____/|_| |_| \__,_|


Logging in to balena-cloud.com
? How would you like to login? Credentials
? Email:
```
At that prompt the CLI freezes and not even CTRL-C works. It's necessary to open another shell prompt and issue `kill -15 PID` for the CLI to exit.

Originally reported at: https://forums.balena.io/t/balena-cli-hangs-when-using-credentials-v11-3-3/16911/

The `npm` installation does not seem to be affected, nor CLI versions 10.x.x.

**Comment by pdcastro** [comment]
Known workarounds:
* Instead of using the interactive menu, pass the credentials on the command line:
`balena login --credentials --email johndoe@gmail.com --password secret`
* Standalone zip for [CLI version 10.17.5](https://github.com/balena-io/balena-cli/releases/tag/v10.17.5) (latest on 10.x.x)
* [npm installation](https://github.com/balena-io/balena-cli/blob/master/INSTALL.md#npm-installation) (any version)

**Comment by pdcastro** [comment]
This appears to have been fixed in balena CLI v11.3.4, and the issue appears to have affected CLI versions 11.3.2 and 11.3.3 only (standalone zip and executable installers). The odd thing is that no specific action was taken to fix it in v11.3.4, which most likely means it that it was a 3rd-party dependency that had broken and got fixed. In which case, I think the most pressing issue to address next is #1293 [Commit a 'npm-shrinkwrap.json' file to control dependency updates](https://github.com/balena-io/balena-cli/issues/1293).
